pulumi/.pre-commit-config.yaml

```
@@ -0,0 +1,22 @@
+        - id: check-yaml
+        - id: check-json
+        - id: check-merge-conflict
+        - id: detect-private-key
+        - id: forbid-binary
+        - id: check-added-large-files
+  -   repo: https://github.com/psf/black-pre-commit-mirror
+      rev: 24.10.0
+      hooks:
+        - id: black
+          language_version: python3.9
+  -   repo: https://github.com/pycqa/flake8
```

yamllint

[error] too many spaces after hyphen

(hyphens)